File: frontend/scripts/check_style.py
import asyncio
from playwright.async_api import async_playwright

async def check():
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        page = await browser.new_page()
        await page.goto('http://localhost:3000/dashboard')
        await asyncio.sleep(2)
        
        halo = await page.query_selector('text="Halo"')
        if halo:
            style = await page.evaluate('''el => {
                const s = window.getComputedStyle(el);
                return `opacity: ${s.opacity}, display: ${s.display}, color: ${s.color}, font: ${s.fontFamily}, zIndex: ${s.zIndex}`;
            }''', halo)
            print("Halo style:", style)
            bbox = await halo.bounding_box()
            print("Halo bbox:", bbox)
        else:
            print("Halo not found")

        header = await page.query_selector('h1:has-text("Dashboard")')
        if header:
            style = await page.evaluate('''el => {
                const s = window.getComputedStyle(el);
                return `opacity: ${s.opacity}, display: ${s.display}, color: ${s.color}`;
            }''', header)
            print("Header style:", style)
            bbox = await header.bounding_box()
            print("Header bbox:", bbox)
        else:
            print("Header not found")

        await browser.close()

if __name__ == '__main__':
    asyncio.run(check())

File: frontend/check_style.js
const { chromium } = require('playwright');
(async () => {
    const browser = await chromium.launch();
    const page = await browser.newPage();
    await page.goto('http://localhost:3000/dashboard');
    await page.waitForTimeout(2000);
    const el = await page.$(':text("Halo")');
    if (el) {
        const style = await el.evaluate(e => {
            const s = window.getComputedStyle(e);
            return {
                display: s.display,
                opacity: s.opacity,
                visibility: s.visibility,
                color: s.color,
                width: s.width,
                height: s.height,
                zIndex: s.zIndex,
                fontFamily: s.fontFamily,
                position: s.position
            };
        });
        console.log("Halo Text Style:", style);
    } else {
        console.log('Halo Text Not found');
    }

    const header = await page.$('text=Dashboard');
    if (header) {
        const hStyle = await header.evaluate(e => {
            const s = window.getComputedStyle(e);
            return {
                display: s.display,
                color: s.color,
                opacity: s.opacity,
                visibility: s.visibility,
                width: s.width,
                height: s.height
            }
        });
        console.log("TopHeader text style:", hStyle);
    } else {
        console.log('TopHeader not found');
    }

    await browser.close();
})();
